src/bin/www.ts

```diff
@@ -0,0 +1,20 @@
+import Koa = require('koa');
+import {productsRouter} from '../routers/product-router';
+import bodyParser = require('koa-bodyparser');
+import {loggingMiddleware} from '../middlewares/logging-middleware';
+import {config} from '../../configurations/configuration';
+import {errorHandlerMiddleware} from '../middlewares/error-handler';
+import {connectToDB} from '../actions/db-connector';
+
+
+const app = new Koa();
+connectToDB();
+
+app
+    .use(loggingMiddleware)
+    .use(errorHandlerMiddleware)
+    .use(bodyParser())
```

Comment:
add external middlewares first and then yours

Reply:
moved the bodyparser to the top